docs/one-click-integrations/github-integration.md

```diff
@@ -115,12 +140,26 @@ The table below lists the data that the GitHub integration collects from your Gi
     <td>
         <p>Deployments:</p>
         <ul>
-            <li><code>deploy_id</code>: pull request ID</li>
+            <li><code>deploy_id</code>: pull request identifier</li>
+            <li><code>timestamp_created</code>: pull request merged date</li>
```

Comment:
See [this comment](https://github.com/codacy/pulse-user-docs/pull/195/files#r1108223461) further down

```suggestion
            <li><code>timestamp_created</code>: merge date of the pull request</li>
```